app/components/embed/footer_component.html.erb

```diff
@@ -11,14 +11,15 @@
     <% end %>
 
     <% if viewer.show_download? %>
+      <% file_count = viewer.purl_object.downloadable_files.length %>
+      <% download_file_label = 'No files available for download' %>
+      <% download_file_label = "#{file_count} #{'file'.pluralize(file_count)} available for download" if file_count > 0 %>
```

Comment:
Would you mind moving this code to the `.rb` part of the component?

Reply:
And maybe consider using the i18n pluralization tool for doing this: https://guides.rubyonrails.org/i18n.html#pluralization

Reply:
@jcoyne added a commit to use i18n, good suggestion, I'm happy to address any further concerns/suggestions.